matrix/common/constants.py

```diff
@@ -49,10 +49,10 @@ class MetadataSchemaName(Enum):
     ORGANOID = "organoid"
 
 
-DEFAULT_FIELDS = ["cell.cell_suspension_id", "cell.genes_detected", "cell.file_uuid",
+DEFAULT_FIELDS = ["cell.cellsuspensionkey", "cell.genes_detected", "cell.file_uuid",
                   "cell.file_version", "cell.total_umis", "cell.emptydrops_is_cell",
-                  "cell.barcode", "specimen.*", "library_preparation.*", "project.*",
-                  "analysis.*"]
+                  "cell.barcode", "cell_suspension.*", "specimen.*", "donor.*",
```

Comment:
Will this include both `cell.cellsuspensionkey` and `cell_suspension.cellsuspensionkey` in the matrix? I suppose it won't cause confusion but unnecessary.

Reply:
Oh yeah it will. I'll take out the extra one.